gallery_dl/extractor/tiktok.py

```diff
@@ -55,14 +55,8 @@ def items(self):
 
             if not downloaded_avatar:
                 avatar_url = post["author"]["avatarLarger"]
-                avatar = text.nameext_from_url(avatar_url, post.copy())
-                avatar.update({
-                    "type"  : "avatar",
-                    "title" : "@" + user,
-                    "id"    : author["id"],
-                    "img_id": avatar["filename"].partition("~")[0],
-                    "num"   : 0,
-                })
+                avatar = \
+                    self._generate_avatar(avatar_url, post, user, author["id"])
```

Comment:
```suggestion
                avatar = self._generate_avatar(
                    author["avatarLarger"], post, user, author["id"])
```
and remove the `avatar_url = post["author"]["avatarLarger"]` line